Sources/GitLabSwift/Core/Support/GLModels.swift

```diff
@@ -138,7 +138,7 @@ public extension GLModel {
     
     struct Project: Decodable {
         public let id: Int
-        public let description: String
+        public let description?: String
```

Comment:
This would need to be `public let description: String?`

Reply:
Thanks, used the typescript syntax for that....